resource-definitions/terraform-driver/credentials/aws-environment-credentials.yaml

```diff
@@ -1,15 +1,17 @@
 apiVersion: entity.humanitec.io/v1b1
 kind: Definition
 metadata:
-  id: aws-file-credentials
+  id: aws-environment-credentials
 entity:
+  # Use the same account as the config we're ma
+  driver_account: ${resources['config.default#aws-account'].account}
   driver_inputs:
-    secret_refs:
-      files:
-        aws_creds:
-          # Using the "value" element of a secret reference to write a value.
-          value: ${resources.config#aws-account.outputs.credentials_file}
     values:
+      credentials_config:
+        environment:
```

Comment:
@docktorrr that's the approach I suggest in [my comment](https://github.com/humanitec-architecture/example-library/pull/11#discussion_r1500658065).